doc/ref_arch/kubernetes/chapters/chapter01.rst

```diff
@@ -41,12 +41,12 @@ Kubernetes-based workloads, and Kubernetes cluster lifecycle management.
 Terminology
 ~~~~~~~~~~~
 
-For terminology used in this document refer to the :doc:`common/glossary:glossary`.
+For terminology used in this document refer to the :ref:`common/glossary:glossary`.
```

Comment:
```suggestion
For terminology used in this document refer to the :ref:`common/glossary`.
```

Reply:
for some reason
```
:ref:`common/glossary`
```
(without `:section`) doesn't build, but
```
:doc:`common/glossary`
```
does